pkgs/by-name/ya/yazi/plugins/recycle-bin/default.nix

```
@@ -0,0 +1,24 @@
+{
+  lib,
+  fetchFromGitHub,
+  mkYaziPlugin,
+}:
+mkYaziPlugin {
+  pname = "recycle-bin.yazi";
+  version = "1.0.0-unstable-2025-08-20";
+
+  src = fetchFromGitHub {
+    owner = "uhs-robert";
+    repo = "recycle-bin.yazi";
+    rev = "3f36069567b4602f841f2377c5f182f9a2480dea";
+    hash = "sha256-1z92wbadcnljj019b1r89l38zyd0nnpm22m269502drbj65grlkz";
```

Are you sure this hash is correct? Try removing it, like this:

```suggestion
    hash = "";
```
Then, save the file and run the build again. It will fail, but Nix will give you the correct hash, which you can then paste here.